ngrok-java/src/main/java/com/ngrok/Listener.java

```diff
@@ -0,0 +1,50 @@
+package com.ngrok;
+
+import java.io.IOException;
+
+/**
+ *
```

Comment:
Should this get a description?

Reply:
Gave it some effort, lemme know what you think 🤷